GCRCatalogs/dc2_coadd.py

```diff
@@ -0,0 +1,130 @@
+"""
+DC2 Coadd Catalog Reader
+"""
+
+import os
+import re
+import warnings
+import numpy as np
+import tables
+import pandas as pd
+from GCR import BaseGenericCatalog
+
+__all__ = ['DC2CoaddCatalog']
+
+class DC2CoaddCatalog(BaseGenericCatalog):
+    r"""
+    DC2 Coadd Catalog reader
+
+    Parameters
+    ----------
+    base_dir: str
+    filename_pattern: str, optional (default: r'merged_tract_\d+\.hdf5')
+    groupname_pattern: str, optional (default: r'coadd_\d+_\d\d$')
+    use_cache : bool, optional (default: True)
+    """
+
+    _native_filter_quantities = {'tract', 'patch'}
+
+    def _subclass_init(self, **kwargs):
+        self._base_dir = kwargs.get('base_dir', '/global/projecta/projectdirs/lsst/global/in2p3/Run1.1/summary')
```

Comment:
This line defines the default base directory, however you also have a base directory specified in the .yaml file. It would be simpler to maintain if the reader only used one or the other.